src/app/shared/services/broker.service.ts

```diff
@@ -74,23 +74,49 @@ export class BrokerService {
     };
   }
 
+  importProjectUsingGeo(accession: string): Observable<any> {
```

Comment:
Why is this any?

Reply:
now replaced with ` Observable<{ project_uuid: string}>`, this is the broker service success response object for this endpoint.

Reply:
I'm hesitant to define an interface/object for it as the broker service responses can vary depending on the endpoint result.